### ISSUE TITLE
[enhancement/consideration] mult-canvas vega view

### ISSUE THREAD

**Issue by nyurik** [vega/vega#916]
For some graphs, it may make sense to be able to create graphs that reside on more than one view. Alternatively, this may be two graphs that share everything except for marks. Usage scenarios:
* multiple layers for leaflet -- one for the chart that moves with the underlying map, another is for the legend or control signals that stays fixed. If implemented as a single chart, moving the map around requires repaints on each frame so that the legend stays in the same location - making it very slow.
* some very long (tall) chart, shown in a scroll-able container, and a separate box on the side for the legend.

**Comment by jheer** [comment]
Yes, this could be interesting and we've thought about related ideas in the past. However, this would be a substantial amount of work to support from a single spec and not on the short- or mid-term roadmap. An alternative option (likely less satisfying) is to use multiple Vega views explicitly.

**Comment by nyurik** [comment]
Agree, nothing for the short term...  I wonder if it would make sense to implement some host-specific signal two-way bindings between multiple instances

**Comment by abudaan** [comment]
Meanwhile this might be useful?

https://github.com/abudaan/vega-multi-view

**Comment by domoritz** [comment]
@abudaan That is amazing. Do you think you could add support for Vega-Lite and send us a PR so we can advertise your library at https://vega.github.io/vega-lite/applications.html?

**Comment by abudaan** [comment]
That should be very well possible I think, I will look into it the coming days.

At the moment I am implementing 2-way data manipulation between Vega instances, see this [prototype](http://app3.bigdator.nl/) to get an idea.

**Comment by domoritz** [comment]
That would be amazing to have in combination with Vega-Lite @kanitw.

I can also see this binding working well in Idyll @mathisonian.
